lib/msal-angular/src/msal.interceptor.ts

```diff
@@ -207,12 +207,16 @@ export class MsalInterceptor implements HttpInterceptor {
                     // Ensure methods being compared are normalized
                     const normalizedRequestMethod = httpMethod.toLowerCase();
                     const normalizedResourceMethod = entry.httpMethod.toLowerCase();
-
                     // Method in protectedResourceMap matches request http method
                     if (normalizedResourceMethod === normalizedRequestMethod) {
-                        entry.scopes.forEach(scope => {
-                            scopesForEndpoint.push(scope);
-                        });
+                        // Validate if scopes comes null to unprotect the resource in a certain http method 
+                        if (entry.scopes === null) {
+                            allMatchedScopes.push(null);
+                          } else {
+                            entry.scopes.forEach((scope) => {
+                              scopesForEndpoint.push(scope);
+                            });
+                          }
```

Comment:
Please dedent to match indendation of `if` block.

Reply:
Done